packages/visx-xychart/src/components/XYChart.tsx

```diff
@@ -118,19 +152,28 @@ export default function XYChart<
     );
   }
 
+  const pointerEventEmitters = usePointerEventEmitters({ source: XYCHART_EVENT_SOURCE });
+  const pointerEventHandlers = usePointerEventHandlers({
+    dataKey: pointerEvents === 'nearest' ? POINTER_EVENTS_NEAREST : POINTER_EVENTS_ALL,
+    onPointerMove,
+    onPointerOut,
+    onPointerUp,
+  });
+  useEventEmitter('pointermove', pointerEventHandlers.onPointerMove, eventSourceSubscriptions);
```

Comment:
`useEventEmitter()` x3 calls are duplicated in all Series, I could push this into `usePointerEventHandlers`